components/new-member-list-item/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from './new-member-list-item.module.scss';
+import { motion } from 'framer-motion';
+
+const NewMemberListItem = ({ newMemberDetails }) => {
+  const { username } = newMemberDetails;
+  const fullName = `${newMemberDetails.first_name + ' ' + newMemberDetails.last_name}`;
+
+  const brokenImageHandler = (e) => {
+    e.target.src = '/images/Avatar.png';
+  };
+
+  return (
+    <div className={classNames.container}>
+      <motion.img
+        layoutId={username}
+        src={newMemberDetails.img_url + `?${Math.random() * 100}`}
+        onError={brokenImageHandler}
+        className={classNames.imgContainer}
+        alt={username}
+      />
+      <h2 className={classNames.nameOfPerson}>
+        {fullName.length > 1
+          ? `${newMemberDetails['first_name']} ${newMemberDetails['last_name']}`
+          : username}
+      </h2>
+    </div>
+  );
+};
+
+NewMemberListItem.propTypes = {
+  newMemberDetails: PropTypes.object
+};
+
+NewMemberListItem.defaultProps = {
+  newMemberDetails: {}
+};
+
+export default NewMemberListItem;
```

Comment:
Also, the same code is being repeated on page `components/member-list-item/index.js`
can we shift the logic to a common component and then use this component accordingly?

Reply:
###### you are talking about this part

#### components/new-member-list-item/index.js

![image](https://user-images.githubusercontent.com/45538392/118702669-d799f980-b832-11eb-8c0f-9cce6823d670.png)

#### components/member-list-item/index.js

![image](https://user-images.githubusercontent.com/45538392/118702323-6c502780-b832-11eb-8235-ad56577f952c.png)

Reply:
## you are talking about this part

#### components/new-member-list-item/index.js

![image](https://user-images.githubusercontent.com/45538392/118702669-d799f980-b832-11eb-8c0f-9cce6823d670.png)

#### components/member-list-item/index.js

![image](https://user-images.githubusercontent.com/45538392/118702323-6c502780-b832-11eb-8235-ad56577f952c.png)